testdata/invalidlineage/second-schema-versionless.txtar

```diff
@@ -31,9 +31,4 @@ lenses: [
 	}
 ]
 -- out/bindfail --
-_schemasAreOrdered.1: _schemasAreOrdered.1: conflicting values true and false:
-    ../../lineage.cue:113:3
-    ../../lineage.cue:114:4
-    ../../lineage.cue:119:5
-    ../../lineage.cue:120:11
-    ./in.cue:4:1
+schema 0.0 must be backwards incompatible with schema 0.0: introduce a breaking change, or redeclare as version 0.1
```

Comment:
hah, this is a funny side effect of using `[0, 0]` as a default value in `#SchemaDef`